midterm_project/midterm_project_victor_razvan/shopping.py

```diff
@@ -0,0 +1,164 @@
+class PrittyPrinterAndMoreMixin:
```

Comment:
Nice choice, but please find a more suitable name or split the two parts into two Mixins